src/fetch/dune.py

```diff
@@ -49,12 +49,17 @@ def _parameterized_query(
         return query_data.with_params(params)
 
     def _get_query_results(
-        self, query: Query, job_id: Optional[str] = None
+        self,
+        query: Query,
+        job_id: Optional[str] = None,
+        performance: str = "medium",
```

Comment:
In my opinion since this is business critical we should be defaulting to `large`. What do you think?
```suggestion
        performance: str = "large",
```

Reply:
Yes I like it. In that case we can set the default performance when instantiating the client. I'll move the parameter to the appropriate place.